src/main/java/net/mcreator/plugin/Plugin.java

```diff
@@ -131,6 +136,18 @@ public String getPluginVersion() {
 		return info.getVersion();
 	}
 
+	public URL toURL() throws MalformedURLException {
+		if (file.isDirectory()) {
+			return (file.toURI().toURL());
+		} else {
+			return (new URL("jar:file:" + file.getAbsolutePath() + "!/"));

```

Comment:
Are those parentheses (before `new URL(...)`) really useful? I don't think we use them in other places of the code.

Reply:
It is used twice in pluginloader, so this is how clean code is done. Code used more than once is put into method instead of inlining it all over the place